jujugui/static/gui/src/app/components/help/tour/_tour.scss

```diff
@@ -0,0 +1,130 @@
+.tour {
+  $tour-max-width: 850px;
+
+  &.lightbox {
+    @media screen and (min-width: $tour-max-width) {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+  }
+
+  .lightbox {
+    &__close {
+      @media screen and (max-width: $tour-max-width - 1) {
+        top: .5rem;
+        right: .5rem;
+      }
+    }
+
+    &__wrapper {
+      text-align: left;
+
+      @media screen and (min-width: $tour-max-width) {
+        width: 850px;
+        left: auto;
+        right: auto;
+        top: auto;
+        bottom: auto;
+      }
+
+      @media screen and (max-width: $tour-max-width - 1) {
+        top: 2.5rem;
+        left: 0;
+        right: 0;
+        bottom: 2.5rem;
+      }
+    }
+
+    &__content {
+      padding: 0;
+
+      @media screen and (min-width: $tour-max-width) {
+        height: 530px;
+      }
+    }
+
+    &__slide {
+      &:nth-child(2) {
```

Comment:
Are these two slides special because of the dimensions of the images? I feel there are better ways to vertically center the content.

Reply:
They're not vertically centred to the containing div, they've vertically centred to the remaining space after the absolutely positioned content of variable height has taken it's cut, unless it's a taller image, in that case, it's fine for the absolute content to overlap the bottom of the image.